### ISSUE TITLE
ME stocking Input Hatch 

### ISSUE THREAD

**Issue by adk95** [MindBrain27/Sky-Greg#77]
![image](https://github.com/MindBrain27/Sky-Greg/assets/50649069/7235da3c-fe6c-411d-b358-a71c0fa0ef89)

cannot select gas type while dragging and dropping 
the UI is invisible behind the name

**Comment by adk95** [comment]
i figured how to put liquid in there by using a bucket
but when i set the circuit to 2 and use a stocking input bus , i used alumium dust and on circuit 2 it does not work selecting circuit 1 works without the gas in electronic blast furnace

![image](https://github.com/MindBrain27/Sky-Greg/assets/50649069/e8755aa0-8279-403a-815d-800284ce76d8)

![image](https://github.com/MindBrain27/Sky-Greg/assets/50649069/331a56da-90d4-4957-af17-16d05bcd9da9)

**Comment by MindBrain27** [comment]
I was trying to get this to work as well but couldn't but if you say it works with just the Aluminium Dust but not the fluids then it leads me to believe it's just the fluid hatch that isn't working correctly.

**Comment by RailGvn** [comment]
> I was trying to get this to work as well but couldn't but if you say it works with just the Aluminium Dust but not the fluids then it leads me to believe it's just the fluid hatch that isn't working correctly.

It seems like the multiblocks just aren't reading the fluids properly from the stocking hatch, FYI I've tried it on multiple other multiblocks like the desulfurizer, large distillery etc. Tried it as the sole fluid input block as well, the recipe just doesn't start.
Interestingly, I've also tried having one normal input hatch with some input in it, and once the recipe is initiated on the multiblock, it starts reading the stocking hatch until whatever is in there depletes. Hope this helps get this fixed.
